airflow/providers/cncf/kubernetes/executors/kubernetes_executor.py

```diff
@@ -224,6 +224,36 @@ def clear_not_launched_queued_tasks(self, session: Session = NEW_SESSION) -> Non
             if time.time() - timestamp > allowed_age:
                 del self.last_handled[key]
 
+        if len(queued_tis) == 0:
+            return
+
+        # airflow worker label selector batch call
+        kwargs = dict(label_selector=f"airflow-worker={self._make_safe_label_value(str(self.job_id))}")
+        if self.kube_config.kube_client_request_args:
+            kwargs.update(**self.kube_config.kube_client_request_args)
```

Comment:
```suggestion
            kwargs.update(self.kube_config.kube_client_request_args)
```

`dict.update()` can take a dict directly without unpacking.